src/test/regress/expected/citus_split_shard_by_split_points_deferred_drop.out

```diff
@@ -61,65 +62,53 @@ SELECT pg_catalog.citus_split_shard_by_split_points(
 
 (1 row)
 
--- The original shards are marked for deferred drop with policy_type = 2.
+-- The original shard is marked for deferred drop with policy_type = 2.
+-- The previous shard should be dropped at the beginning of the second split call
 SELECT * from pg_dist_cleanup;
- record_id | operation_id | object_type |          object_name          | node_group_id | policy_type
+ record_id | operation_id | object_type |                               object_name                                | node_group_id | policy_type
 ---------------------------------------------------------------------
-        11 |          777 |           1 | public.table_to_split_8981000 |            14 |           2
-        12 |          778 |           1 | public.table_to_split_8981001 |            16 |           2
-(2 rows)
+       512 |          778 |           1 | citus_split_shard_by_split_points_deferred_schema.table_to_split_8981001 |            16 |           2
+(1 row)
 
--- The physical shards should not be deleted.
+-- One of the physical shards should not be deleted, the other one should.
 \c - - - :worker_1_port
-SELECT relname FROM pg_class where relname LIKE '%table_to_split_%' AND relkind = 'r';
+SELECT relname FROM pg_class where relname LIKE '%table_to_split_%' AND relkind = 'r' ORDER BY relname;
         relname
 ---------------------------------------------------------------------
- table_to_split_8981000
  table_to_split_9999000
  table_to_split_9999002
-(3 rows)
+(2 rows)
 
 \c - - - :worker_2_port
-SELECT relname FROM pg_class where relname LIKE '%table_to_split_%' AND relkind = 'r';
+SELECT relname FROM pg_class where relname LIKE '%table_to_split_%' AND relkind = 'r' ORDER BY relname;
         relname
 ---------------------------------------------------------------------
  table_to_split_8981001
  table_to_split_9999001
  table_to_split_9999003
 (3 rows)
 
--- Set a very short(1ms) time interval to force deferred drop cleanup.
+-- Perform deferred drop cleanup.
 \c - postgres - :master_port
-ALTER SYSTEM SET citus.defer_shard_delete_interval TO 1;
-SELECT pg_reload_conf();
- pg_reload_conf
----------------------------------------------------------------------
- t
-(1 row)
-
--- Give enough time for the deferred drop cleanup to run.
-SELECT pg_sleep(2);
- pg_sleep
----------------------------------------------------------------------
-
-(1 row)
-
```

Comment:
was this flaky? If so maybe increase pg_sleep instead and run `ALTER SYSTEM SET defer_shard_delete_interval = -1` to reset. Because this seems to explicitly test that them maintenance daemon is doing its job.

Reply:
actually never mind it seems like this output was not even matching the actual test file. So you can keep it as is.

Reply:
No, it was just a wrong output. These lines were modified at some point in the sql file but the output file wasn't updated accordingly. We haven't noticed that because we don't run this test in any schedule:)

So this change is not because of flakiness, but just correcting the expected output.

Reply:
your last message just showed up. Yeah, that's the case